public/templates/window/action_manager.hbs

```diff
@@ -3,28 +3,30 @@
   style="width:{{position.width}}px; height:{{position.height}}px; top:{{position.top}}px; left:{{position.left}}px;">
   <div class="action-label">
     <div>{{name}}</div>
+    {{#if clickable}}
     <div style="right: 0px;position: absolute;"><a id="action-manager-reset" class="mdi mdi-restore"></a></div>
+    {{/if}}
   </div>
   <div style="display:inline-flex;">
     <i id="action-manager-drag" class="mdi mdi-drag"></i>
     <div class="action-bar">
-      <a class="action {{#if actions.protocol}}activation-protocol{{/if}}" data-action="protocol">
+      <a class="action {{#if actions.protocol}}lancer-protocol{{/if}}" data-action="protocol">
```

Comment:
I believe these classes like `activation-protocol`, `activation-full`, etc. (`activation-<action>`) are not defined, this is probably from old code.

May be worth grepping for other occurrences of these elsewhere in the code to clean up.

Reply:
I removed the other occurrences of these when fixing the sheet action buttons.